app/app.component.html

```diff
@@ -1,136 +1,114 @@
-<div class="menu-side-bar" (click)="onBodyClick($event)" [hidden]="!isNavigationMenuVisible">
-  <div class="overbody-container"></div>
-  <div class="close-button icon-ic-close" (click)="isNavigationMenuVisible = false"></div>
-  <div class="menu-container table">
-    <div class="table-row">
-      <div class="nav-menu table-cell">
-        <div class="nav-menu-icon-container"><i class="icon-ic-just-arrived-logo"></i></div>
-        <div class="nav-menu-item-container">
-          <div class="nav-menu-item clickable" *ngIf="user">
-            <a class="nav-menu-link" routerLink="/my-jobs">
-              <i class="icon-ic-my-jobs"></i>
-              {{'menu.main.my_assignment' | translate}}
-            </a>
+<div class="navbar-fixed-top" [ngStyle]="(isNavigationMenuVisible || isLanguageMenuVisible) && {'height': '100%'}">
```

Comment:
Wrong syntax and mess understandable logic. This should be with ngClass and to apply in this condition.
https://angular.io/docs/ts/latest/api/common/index/NgClass-directive.html